athena/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "apply_resource" {
+  description = "use to conditionally apply resource from the calling module"
+  default     = true
```

Comment:
What's the use case for this conditional? When would you want to use this module and _not_ create the resources?

Reply:
Just looked at the [other Athena PR](https://github.com/nationalarchives/tdr-aws-accounts/pull/17/files#diff-0bc768eaa6253864050b511f5907053cR95) and I see the use case for when you want to create the resource only in some environments. 👍 

Excitingly, it looks like Terraform might be about to release a [better solution](https://github.com/hashicorp/terraform/issues/17519#issuecomment-632403492) which will mean we can get rid of all these `apply_resource` varibles. I'll add a card to the backlog.

Reply:
yes, it will be good to get rid of this workaround